versions/3.0.md

```diff
@@ -113,12 +113,14 @@ Field Name | Type | Description
 <a name="oasSchemes"></a>schemes | [`string`] | The transfer protocol of the API. Values MUST be from the list: `"http"`, `"https"`, `"ws"`, `"wss"`. If the `schemes` is not included, the default scheme to be used is the one used to access the OpenAPI definition itself.
 <a name="oasConsumes"></a>consumes | [`string`] | A list of MIME types the APIs can consume. This is global to all APIs but can be overridden on specific API calls. Value MUST be as described under [Mime Types](#mimeTypes).
 <a name="oasProduces"></a>produces | [`string`] | A list of MIME types the APIs can produce. This is global to all APIs but can be overridden on specific API calls. Value MUST be as described under [Mime Types](#mimeTypes).
+<a name="oasResponses"></a>responses | [Responses]
 <a name="oasPaths"></a>paths | [Paths Object](#pathsObject) | **Required.** The available paths and operations for the API.
-<a name="oasDefinitions"></a>definitions | [Definitions Object](#definitionsObject) | An object to hold data types produced and consumed by operations.
+<a name="oasSchemas"></a>schemas | [Schemas Object](#schemasObject) | An element to hold various schemas for the specification.
```

Comment:
As mentioned in #563, I would prefer definitions/schemas to schemas/definitions, though I can understand resistance to reusing a property name from a previous version with a different name. 

For machine readers, the distinction should be clear from the version property, but humans might be confused. On the other hand, `schemas` on the top level might lead to confusion with JSON schema (i.e. that everything in it is a JSON schema document), which only applies to the model definitions, while actually everything in here is a definition of some kind, so `definitions` actually captures it better.

`global` might be read to indicate that the contents (specifically parameters) are valid for every operation.

`library` doesn't cause confusion (I think), but also doesn't really say what it is.

Unfortunately I have no other ideas here.